src/upload/system/library/transbank/composer.json

```
@@ -1,6 +1,12 @@
 {
     "require": {
-        "transbank/transbank-sdk": "~2.0"
+        "transbank/transbank-sdk": "~2.0",
+        "monolog/monolog": "^1.27"
+    },
+    "autoload": {
+        "psr-4": {
+            "Transbank\\Utils\\": "system/library/Transbank/utils/"
```

maybe this is better, is a descriptive namespace for the pluginfiles
```suggestion
            ""Transbank\\Opencart\\Webpay\\Utils\\": "system/library/Transbank/utils/"
```
change the folder name to `Utils`

---

The corresponding modifications will be made.